src/devtools/client/debugger/src/reducers/source-actors.js

```diff
@@ -198,8 +216,11 @@ export function getAllThreadsBySource(state) {
   return queryThreadsBySourceObject(state.sourceActors);
 }
 
-export function getSourceActorBreakpointHitCounts(state, id) {
-  const { breakpointHitCounts } = getResource(state.sourceActors, id);
+export function getSourceActorBreakpointHitCounts(state, id, lineNumber) {
+  const { breakpointHitCounts, min, max } = getResource(state.sourceActors, id);
+  if (lineNumber && (!breakpointHitCounts || max < lineNumber || min > lineNumber)) {
```

Comment:
Is it possible for `lineNumber` to be falsy?

Reply:
I think it *was* when I started writing this and now it's not. Let me double check that.

Reply:
Removed!